Jenkinsfile.nightly

```diff
@@ -0,0 +1,92 @@
+pipeline {
+    agent { label 'go'}
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
+        timeout(time: 90, unit: 'MINUTES')
+    }
+    environment {
+        WORKING_DIR = "/home/jenkins/go/src/github.com/kiegroup/kogito-cloud-operator/"
+        GOPATH = "/home/jenkins/go"
+        GOCACHE = "/home/jenkins/go/.cache/go-build"
+    }
+    stages {
```

Comment:
where is the logic that will make sure that this Job will only push images **if** there where new PR merged during the day?

Reply:
This kind of logic you did directly in the Jenkins job (manual configuration) I will document this as well.